### PR TITLE
VillageHQ Refactor

### PR DESCRIPTION
**Pull request by aumu1031**

### CODE REVIEW

File: ui_components/src/village/StrategicInfoItem.js
Comment (aumu1031) [inline ui_components/src/village/StrategicInfoItem.js:1]: Copied 1:1 from Village.js

File: ui_components/src/village/KageQuarters.js
Comment (aumu1031) [inline ui_components/src/village/KageQuarters.js:1]: Copied 1:1 from Village.js

File: ui_components/src/village/TradeDisplay.js
Comment (aumu1031) [inline ui_components/src/village/TradeDisplay.js:1]: Copied 1:1 from Village.js

File: ui_components/src/village/Village.css
Comment (aumu1031) [inline ui_components/src/village/Village.css:1]: Removed modal CSS, left minor style tweak in from v0.12

File: ui_components/src/village/Village.js
Comment (aumu1031) [inline ui_components/src/village/Village.js:1]: Pulled out all pages and utils into separate files, only change is addition of Prosperity policy data.

File: ui_components/src/village/VillageHQ.js
Comment (aumu1031) [inline ui_components/src/village/VillageHQ.js:1]: Copied 1:1 from Village.js

File: ui_components/src/village/WarTable.js
Comment (aumu1031) [inline ui_components/src/village/WarTable.js:1]: Copied 1:1 from Village.js

File: ui_components/src/village/WorldInfo.js
Comment (aumu1031) [inline ui_components/src/village/WorldInfo.js:1]: Copied 1:1 from Village.js, uses modal and imports StrategicInfoItem

**Review by levimeahan — APPROVED** [review]

**Pull request merged**